druntime/src/core/thread/fiber/package.d

```diff
@@ -153,15 +153,20 @@ package
         version (AsmX86_64_Posix)   {} else
         version (AsmExternal)       {} else
         {
-            // NOTE: The ucontext implementation requires architecture specific
-            //       data definitions to operate so testing for it must be done
-            //       by checking for the existence of ucontext_t rather than by
-            //       a version identifier.  Please note that this is considered
-            //       an obsolescent feature according to the POSIX spec, so a
-            //       custom solution is still preferred.
-            import core.sys.posix.ucontext : getcontext, makecontext, MINSIGSTKSZ, swapcontext, ucontext_t;
+            version = ucontext_Posix;
         }
     }
+
+    version (ucontext_Posix)
+    {
+        // NOTE: The ucontext implementation requires architecture specific
+        //       data definitions to operate so testing for it must be done
+        //       by checking for the existence of ucontext_t.
```

Comment:
Same as before. Please rephrase this comment.  As the "checking for the existence of ucontext_t" no longer applies since #20632.

Other than that. Should be OK for x86_64+CET.

Reply:
Oops, yes. I removed this explanation completely because it is outdated